docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -160,3 +160,15 @@ Columns with filters applied have a blue funnel displayed next to the title.
 {{< figure src="/static/img/docs/tables/filtered-column.png" max-width="500px" caption="Filtered column" class="docs-image--no-shadow" >}}
 
 To remove the filter, click the blue funnel icon and then click **Clear filter**.
+
+## Table footer
+
+You can use the table footer to show [calculations]({{< relref "../../calculation-types/" >}}) on fields.
+
+After enabling the table footer, you can select your **Calculation** and select the **Fields** that should be calculated. Not selecting any field apply the calculation to all numeric fields.
```

Comment:
After you enable the table footer:

1. Select the **Calculation**. 
1. Select the **Fields** you want to calculate. 

The system applies the calculation to all numeric fields if you do not select a field.